docs/general/ddl/fulltext-indices.rst

```diff
@@ -175,6 +175,11 @@ Composite indices can include nested columns within object columns as well::
     ... );
     CREATE OK, 1 row affected (... sec)
 
+.. NOTE::
+
+    If ``plain`` index method is used, this internally translates to
+    ``fulltext with (analyzer = 'keyword')``.
+
```

Comment:
Isn't this kinda an implementation detail. Why document it?

Reply:
I asked around and we agreed it can be weird to create with `plain` and in `show create table` see `fulltext`.

Reply:
But it doesn't?


```
↪  cr8 run-crate latest-nightly -- @crash <<EOF
create table tbl (description text index using plain);
show create table tbl;
EOF
```

```
# @crash
========

CONNECT OK
CREATE OK, 1 row affected (0.121 sec)
+-----------------------------------------------------+
| SHOW CREATE TABLE doc.tbl                           |
+-----------------------------------------------------+
| CREATE TABLE IF NOT EXISTS "doc"."tbl" (            |
|    "description" TEXT                               |
| )                                                   |
| CLUSTERED INTO 4 SHARDS                             |
| WITH (                                              |
...
```

Reply:
It does for composite indexes ...
```sql
cr> CREATE TABLE t01 (a TEXT, b TEXT, INDEX i using PLAIN (a,b));
cr> SHOW CREATE TABLE t01;
+-----------------------------------------------------+
| SHOW CREATE TABLE doc.t01                           |
+-----------------------------------------------------+
| CREATE TABLE IF NOT EXISTS "doc"."t01" (            |
|    "a" TEXT,                                        |
|    "b" TEXT,                                        |
|    INDEX "i" USING FULLTEXT ("a", "b") WITH (       |
|       analyzer = 'keyword'                          |
|    )                                                |
| )                                                   |
+-----------------------------------------------------+
```

Reply:
I see, in that case we may need to be even more verbose, because this may give the impression that `PLAIN` is generally equivalent to using `FULLTEXT (...) WITH (analyzer = 'keyword')` but that's not the case.
It's more that `PLAIN` has no effect on a `INDEX` definition.

Any `INDEX` definition will result in an index with where we also store doc frequencies, positions and normalization values. 
A regular column with PLAIN won't have those.

Reply:
Initially I had this PR: https://github.com/crate/crate/pull/14543 so that we don't even allow to use `PLAIN` for such indices and avoid any confustion, what do you think?

Reply:
Isn't it better (less confusing) to completely forbid `plain` named indices?

Reply:
I think it would be okay to make it more strict/forbid it, but only for 5.5.0, not as a hotfix.
And it would have to cover any `INDEX` definition, not just with more than one source.